mdata/cache/sort_uint32.go

```diff
@@ -0,0 +1,7 @@
+package cache
+
+type sortableUint32 []uint32
```

Comment:
Ok, will do that. although i think it's slightly annoying that `sort.Sort()` has no `reverse` parameter, which means if we want to sort in two directions we'll need two sortable types.